ci-connect-cli/helm/test-service/values.yaml

```diff
@@ -0,0 +1 @@
+test
```

Comment:
Can remove this file without the other testing files of `.keptn` it does nothing